### PR TITLE
Updated examples to glm

### PR DESCRIPTION
**Pull request by roymacdonald**
I made a big mistake with git and the easiest way to fix it was to get simply copy all the modified examples over the master and commit and PR again.
@bakercp

### CODE REVIEW

File: examples/math/trigonometryExample/src/ofApp.cpp [hunk 1]
```diff
@@ -35,27 +35,26 @@ void ofApp::setup(){
...
     
     if (!ofGetMousePressed()) {//press the mouse to stop the angle from incrementing.
 		angle+=rotationSpeed;//at each update the angle get's incremented 
 	}
-	if (angle>=TWO_PI) { //if the angle is more than or equal to two PI (a full rotation measured in Radians) then make it zero.
-		angle=0;
-	}
-
+	 //if the angle is more than or equal to two PI (a full rotation measured in Radians) then make it zero.
+	angle = ofWrap(angle, 0, TWO_PI);
```
Comment (bakercp) [inline examples/math/trigonometryExample/src/ofApp.cpp]: This still isn't fixed.  I'll fix it in a PR.

Comment (roymacdonald) [reply examples/math/trigonometryExample/src/ofApp.cpp]: this is fine. What was there before was doing the same but only for when the angle was more than a full rotation but it was not handling the case when it was less than zero, which it is now.

Comment (bakercp) [reply examples/math/trigonometryExample/src/ofApp.cpp]: OK.  Reverting in my PR.

**Comment by roymacdonald** [comment]
seems to be ok then to merge. the CI is passing. @arturoc does the CI compile the examples?
I will double check visually each example though. @bakercp did you visually check the iOS examples?
I'll try yo get android running again to visually check too.

**Comment by bakercp** [comment]
My visual inspection of the iOS examples was successful.  As soon as we figure out the quaternion issue https://github.com/openframeworks/openFrameworks/issues/6327 and confirm android, I think this is ready to go.

**Comment by roymacdonald** [comment]
good, I hope we get the ofQuat issue solved soon. I will check androind over the next days and report back.

**Comment by ofTheo** [comment]
@bakercp is there anything else left? #6327 is closed now.

**Comment by bakercp** [comment]
@ofTheo @roymacdonald  Roy -- did you make the `glm::rotation` changes?  I can't seem to find them in this PR.

**Comment by roymacdonald** [comment]
@bakercp Nope. This function we found that glm has it does not act in the same way as the ofQuaternion. I was testing it while in Denver, I'll look at it again tomorrow. But i think that it is fine to merge all this as-is, and then I make a different PR with the fixed orientation example. I just

**Comment by ofTheo** [comment]
sorry @roymacdonald I think my review requesting changes wasn't public. see above! Thanks!

File: addons/ofxiOS/src/utils/ofxiOSCoreMotion.mm [hunk 2]
```diff
@@ -54,12 +54,12 @@
     [motionManager stopDeviceMotionUpdates];
     enableAttitude = false;
     roll = pitch = yaw = 0;
-    attitudeQuat.set(0, 0, 0, 1);
-    rotMatrix.makeIdentityMatrix();
-    gravity.set(0,0,0);
-    userAcceleration.set(0, 0, 0);
-    magneticField.set(0, 0, 0);
-    
+    attitudeQuat = glm::quat();
```
[the remark this answers is not in the document]
Comment (roymacdonald) [reply addons/ofxiOS/src/utils/ofxiOSCoreMotion.mm]: sure.  I see. I don't like this new glm behavior. 
It was @bakercp who updated the iOS stuff but I can check that too.
I'll let you all know once ready for merging.
best

**Comment by ofTheo** [comment]
ps: @roymacdonald 

I think glm::rotation is a 1:1 replacement for ofQuaternion::makeRotate 

Here is a quick example that switches between the two. 
It seems identical to me. Let me know if I am missing something :) 

```
ofEasyCam cam;

//--------------------------------------------------------------
void ofApp::draw(){
    ofEnableLighting();
    ofLight l1;
    l1.setPosition(300, 300, 300);
    l1.enable();
    ofEnableDepthTest();
    
    cam.begin();
    
        ofQuaternion q;
        q.makeRotate(glm::vec3(0, 1, 0), glm::vec3(1, 0, 0));
        
        glm::quat qGLM = q;
        if( ofGetFrameNum() % 10 == 0 ){
            glm::quat qGLM = glm::rotation(glm::vec3(0, 1, 0), glm::vec3(1, 0, 0));
        }
        
        ofMultMatrix( glm::toMat4(qGLM) );
        
        ofDrawCone(glm::vec3(0), 50, 300);
    cam.end();
}
```

**Comment by roymacdonald** [comment]
@ofTheo @bakercp I've updated the orientation example as well as the glm constructor issue that theo mentioned. I don't know if there are more issues with this glm constructor thing.

**Comment by ofTheo** [comment]
Thanks @roymacdonald !
As far as I can tell this looks good to merge.

**Comment by roymacdonald** [comment]
Super. Now on what does it depend to get it merged?

**Comment by arturoc** [comment]
thanks roy!

**Comment by bakercp** [comment]
Epic. Thanks @roymacdonald !

**Comment by ofZach** [comment]
👏👏  thx 👏👏

**Pull request merged**